iterators.tex

```diff
@@ -1069,7 +1070,7 @@
 \rSec2[indirectcallables.as_function_t]{as_function_t type}
 
 \pnum
-The \tcode{\oldtxt{FunctionType}}\tcode{\newtxt{as_function_t}} is an alias used to turn a
+The \tcode{\oldtxt{FunctionType}}\tcode{as_function_t} is an alias used to turn a
```

Comment:
I can't see on my phone. Is this whole section removed? Is that why `\newtxt` is wrong?